src/content/language/dot-shorthand.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Shorthand syntax
+description: Learn about the shorthand syntax in Dart.
```

Comment:
Perhaps we should call it the dot shorthand syntax. There might be other shorthand syntaxes coming down the line from the language team and it might be confusing to have this broader-sounding title when that happens.

Reply:
+1 for `Dart shorthand syntax`